packages/rosaenlg/src/LanguageEnglish.ts

```diff
@@ -144,6 +144,7 @@ export class LanguageEnglish extends LanguageImpl {
       genderOwner: this.genderNumberManager.getRefGender(owner, params),
       numberOwner: this.genderNumberManager.getRefNumber(owner, params),
       numberOwned: null, // we do not care
+      personOwner: 3, //first and second person not implemented

```

Comment:
not really useful as parameter is optional and `getDet` implemtation in `LanguageEnglish.ts` does not use it

Reply:
removed